src/amp-ads/40_Experimental_Ads/Carousel_Lightbox_Ad.html

```diff
@@ -0,0 +1,418 @@
+    <script async custom-element="amp-lightbox" src="https://amp-ads.firebaseapp.com/dist/v0/amp-lightbox-0.1.max.js"></script>
+    <script async src="https://amp-ads.firebaseapp.com/dist/amp-inabox.js"></script>
+    <style amp4ads-boilerplate>body{visibility:hidden}</style>
+    <!-- ## Summary-->
+    <!--
+    Sample AMP Ad using [amp-lightbox](https://www.ampproject.org/docs/reference/components/amp-lightbox),
+    and [amp-carousel](https://www.ampproject.org/docs/reference/components/amp-carousel) to create an interactive ad.
+    -->
+    <!-- ## Styling -->
+    <!--
+    This is an advanced example that requires some styling to make it look and function properly.
+    -->
+    <style amp-custom>
+      .root-container {
+        font-family: 'Roboto', sans-serif;
+        font-size: 12px;
+        line-height: 18px;
+        overflow: hidden;
+        background: #6a594b;
+        color: #ffffff;
+        width: 100%;
+        height: 100vh;
+        position: relative;
+        display: flex;
+        flex-direction: column;
+      }
+      .main-container {
+        flex: 1;
+        display: flex;
+        justify-content: flex-end;
+        flex-direction: column;
+      }
+      .footer {
+        background: #000000;
+        color: #ffffff;
+        padding: 8px;
+        text-transform: uppercase;
+        display: flex;
+        align-items: center;
+      }
+      .footer-logo {
+        border-right: 1px solid #ffffff;
+        padding: 0 8px 0 0;
+        margin: 0 8px 0 0;
+      }
+      .logo-img {
+        display: block;
+      }
+      .stretch {
+        flex: 1;
+      }
+      .button {
+        cursor: pointer;
+        text-decoration: none;
+        text-transform: uppercase;
+        padding: 6px 10px;
+        color: #ffffff;
+        display: inline-block;
+        background-color: #2979ff;
+      }
+      .carousel-item {
+        width: 184px;
+        height: 105px;
+        padding: 12px 2px;
+        color: #000000;
+        position: relative;
+      }
+      .carousel-item:first-child {
+        padding-left: 12px;
+      }
+      .carousel-item:last-child {
+        padding-right: 12px;
+      }
+      .carousel {
+        width: 100%;
+        position: absolute;
+      }
+      .carousel-item-inner {
+        width: 180px;
+        height: 105px;
+        padding: 2px;
+        background: rgba(255, 255, 255, 0.5);
+        border-radius: 4px;
+        position: relative;
+      }
+      .carousel-img {
+        display: block;
+        border: 2px solid #ffffff;
+        border-radius: 2px;
+      }
+      .carousel-button-container {
+        position: absolute;
+        left: 0;
+        right: 0;
+        bottom: 0;
+        padding: 10px;
+        text-align: center;
+      }
+      /* Styling for lightbox */
+      amp-lightbox {
+        font-family: 'Roboto', sans-serif;
+        font-size: 12px;
+        background: rgba(0, 0, 0, 0.5);
```

Comment:
suggestion: make the background darker e.g. 0.9 - otherwise the content shining through is a bit distracting.